docs/writing.md

```diff
@@ -0,0 +1,161 @@
+# Writing documentation
+
+## Conventions
+
+* Write documentation in Markdown (`.md` files) where possible. While Sphinx
```

Comment:
To be simple can we just remove all the rst comments in this doc? Maybe just note rst is supported by sphinx but we're not using it, then just focus on markdown.